branca/colormap.py

```diff
@@ -70,24 +70,27 @@ class ColorMap(MacroElement):
         The right bound of the color scale.
     caption: str
         A caption to draw with the colormap.
+    max_labels : int
+        Maximum number of legend values
```

Comment:
I interpret "values" to mean something different than "ticks". Values are the values used in the colorbar, which can be more than the number of labels or ticks.